examples/nais-max.yaml

```diff
@@ -61,3 +61,14 @@ spec:
   logtransform: http_loglevel # Optional. For accesslogs, set log_level based on http status code
   service:
     port: 80 # default service-port
+  accessPolicy:
+    AllowAllIngress: false # optional, opens up for all inbound access
```

Comment:
Lowercase allow?

To group things logically, could we nest things down a level and say e.g.

```
accessPolicy:
  ingress:
    allowAll: true|false
    rules:
      - application: app1
```

Reply:
Skulle vært liten forbokstav her ja.
Og liker idéen med å gruppere `allowAll` under `ingress` og `egress`.